packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -1442,6 +1442,23 @@ abstract class RenderSliver extends RenderObject {
     return null;
   }
 
+  /// This returns a absolute [Size] of the sliver.
+  ///
+  /// The dimensions are always positive and is only valid after [layout] has
```

Comment:
The dimensions are always positive and calling this is only valid after [layout] has completed.